tests/datasets/test_sentinel.py

```diff
@@ -3,14 +3,98 @@
 
 import os
 from pathlib import Path
+from typing import List
 
 import matplotlib.pyplot as plt
 import pytest
 import torch
 import torch.nn as nn
+from _pytest.fixtures import SubRequest
 from rasterio.crs import CRS
 
-from torchgeo.datasets import BoundingBox, IntersectionDataset, Sentinel2, UnionDataset
+from torchgeo.datasets import (
+    BoundingBox,
+    IntersectionDataset,
+    Sentinel1,
+    Sentinel2,
+    UnionDataset,
+)
+
+
+class TestSentinel1:
+    @pytest.fixture(
+        params=[
+            # Only horizontal or vertical receive
+            ["HH"],
+            ["HV"],
+            ["VV"],
+            ["VH"],
+            # Both horizontal and vertical receive
+            ["HH", "HV"],
+            ["HV", "HH"],
+            ["VV", "VH"],
+            ["VH", "VV"],
```

Comment:
Why are these not just `set(["HH", "HV"])` and `set(["VV", "VH"])`?

Reply:
For our MSI datasets, we want to be able to optionally select a subset of bands (RGB-only, RGB+NIR, etc.). I was following the same approach here, but I'm not actually sure if that makes sense for SAR. Is there any reason why someone might want to only use a single band and not both? If not, I can simplify the code quite a bit.